WordPress/Classes/ViewRelated/Me/App Settings/About/UnifiedAboutViewController.swift

```diff
@@ -6,12 +6,14 @@ struct AboutItem {
     let title: String
     let subtitle: String?
     let cellStyle: AboutItemCellStyle
+    let eventButton: UnifiedAboutEvent.Button?
     let action: (() -> Void)?
 
-    init(title: String, subtitle: String? = nil, cellStyle: AboutItemCellStyle = .default, action: (() -> Void)? = nil) {
+    init(title: String, subtitle: String? = nil, cellStyle: AboutItemCellStyle = .default, eventButton: UnifiedAboutEvent.Button?, action: (() -> Void)? = nil) {
```

Comment:
The new parameter has no default value on purpose.

My rationale is that if a new button is added to this screen, I want to make sure we don't forget to add the corresponding event button.  The developer will have to explicitly set it to `nil` if there's nothing to track.